composer.json

```diff
@@ -9,6 +9,7 @@
     "version": "31",
     "require": {
         "php": ">=7.4",
+        "dealerdirect/phpcodesniffer-composer-installer": "^0.7.2 || ^1.0",
```

Comment:
I believe that `"^1.0"` will be enough

Reply:
There's nothing here that's *not* compatible with `^0.7.0`. I've specifically kept this because I expect other packages may take a while to upgrade / allow the new version. If there's something that requires `^0.7`, it'll work today, but not after merging if we change this to `^1.0` only. Listing both supported versions means that interoperability with other packages is maintained; Composer will pick the "best" version, which is typically the highest mutually-supported version. If we use `^1.0` only here, then we're forcing others to upgrade their packages in order for their coding standard to be used in conjunction with this standard.